skiko/src/awtMain/cpp/windows/directXRedrawer.cc

```diff
@@ -21,10 +21,37 @@
 #include <dxgi1_6.h>
 
 #include <dcomp.h>
-#pragma comment(lib, "dcomp.lib")
 
 const int BuffersCount = 2;
 
+namespace DCompLibrary {
+    static bool isInit = false;
+    static HINSTANCE library = nullptr;
+
+    void load() {
+        if (!isInit) {
+            isInit = true;
+            library = LoadLibrary("dcomp.dll");
+        }
+    }
+
+    HRESULT DCompositionCreateDevice(
```

Comment:
I'd move it to a separate file to possible reuse in swing redrawer or so

Reply:
Done